core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -1370,9 +1370,9 @@ public static int getFilterAndProjectMinOutputPageRowCount(Session session)
         return session.getSystemProperty(FILTER_AND_PROJECT_MIN_OUTPUT_PAGE_ROW_COUNT, Integer.class);
     }
 
-    public static MarkDistinctStrategy markDistinctStrategy(Session session)
+    public static DistinctAggregationsStrategy distinctAggregationsStrategy(Session session)
     {
-        return session.getSystemProperty(MARK_DISTINCT_STRATEGY, MarkDistinctStrategy.class);
+        return session.getSystemProperty(DISTINCT_AGGREGATIONS_STRATEGY, DistinctAggregationsStrategy.class);
```

Comment:
you should merge this commit with the commit that re-adds the MARK_DISTINCT_STRATEGY propertry